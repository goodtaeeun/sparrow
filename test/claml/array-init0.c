int f() {
  char b[] = {};
  return 0;
}
