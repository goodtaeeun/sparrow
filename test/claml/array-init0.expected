/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "array-init0.c"
int f(void) {
  char b[0];

  {
#line 3
    return (0);
  }
}
